lib/sorbet-rails/model_plugins/active_storage_methods.rb

```diff
@@ -0,0 +1,60 @@
+# typed: true
+require ('sorbet-rails/model_plugins/base')
+class SorbetRails::ModelPlugins::ActiveStorageMethods < SorbetRails::ModelPlugins::Base
+  sig { params(model_class: T.class_of(ActiveRecord::Base), available_classes: T::Set[String]).void }
+  def initialize(model_class, available_classes)
+    super
+  end
+
+  sig { implementation.params(root: Parlour::RbiGenerator::Namespace).void }
+  def generate(root)
+    # Check that ActiveStorage the attachment_reflections method exists
+    # It was added in 6.0, so it isn't available for 5.2.
+    return unless defined?(@model_class.attachment_reflections) && @model_class.attachment_reflections.length > 0
+
+    assoc_module_name = self.model_module_name("GeneratedAssociationMethods")
+    assoc_module_rbi = root.create_module(assoc_module_name)
+
+    attachment_reflections = @model_class.attachment_reflections.transform_values { |attachment| attachment.class }
```

Comment:
I guess I need to add `attachment_reflections` to the methods for ActiveRecord::Base in the `activerecord.rbi`?

Reply:
Yeah, if there are sorbet typecheck errors without it